bp02/jcasc/main.yaml

```diff
@@ -5,7 +5,7 @@ unclassified:
   bundleStorageService:
     activated: true
     activeBundle:
-      name: "casc-mm-store"
+      name: "casc-mc-store"
```

Comment:
@carlosrodlop - I updated this to `casc-mc-store`, but please revert this change if it will cause problems.